pallets/portfolio/src/lib.rs

```diff
@@ -869,4 +931,11 @@ impl<T: Config> PortfolioSubTrait<T::AccountId> for Module<T> {
         PortfolioLockedNFT::remove(portfolio_id, (ticker, nft_id));
         Ok(())
     }
+
+    fn skip_portfolio_affirmation(portfolio_id: &PortfolioId, ticker: &Ticker) -> bool {
+        if T::Asset::skip_ticker_affirmation(&portfolio_id.did, ticker) {
+            return true;
+        }
```

Comment:
If we want both asset-level pre-approvals and portfolio-level pre-approvals.  Then we should check if the portfolio has a custodian (the owner has given custodianship control to another identity), before allowing asset-level pre-approvals.  Otherwise there is no point to restrict the portfolio-level pre-approvals manage to the custodian, since the portfolio owner can bypass that restriction.